packages/cozy-procedures/src/components/overview/PersonalDataFullyCompleted.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import snarkdown from 'snarkdown'
+
+import { InlineCard, translate } from 'cozy-ui/transpiled/react'
+
+const capitalize = str =>
+  str.charAt(0).toUpperCase() + str.slice(1).toLowerCase()
+
+const PersonalDataFullyCompleted = ({ navigateTo, personalData, t }) => {
+  const {
+    firstname,
+    lastname,
+    maritalStatus,
+    numberOfDependants,
+    employmentStatus,
+    employmentContract,
+    salary,
+    additionalIncome,
+    propertyStatus,
+    creditsTotalAmount,
+    propertyLoan,
+    address,
+    phone,
+    email
+  } = personalData
+  return (
+    <InlineCard
+      className="u-c-pointer u-p-1 u-flex u-flex-column"
+      onClick={() => navigateTo('personal')}
+    >
+      <p
+        dangerouslySetInnerHTML={{
```

Comment:
Writing it in case in prompts any other ideas: I think it's fine to use `dangerouslySetInnerHTML` here, because the input comes from the user and is never presented to someone else — so the worst case is you XSS yourself.